src/runtime_src/core/edge/drm/zocl/zocl_ioctl.c

```diff
@@ -248,9 +257,23 @@ static int zocl_pcap_download(struct drm_zocl_dev *zdev,
 		data[i+1] = data[i+2];
 		data[i+2] = temp;
 	}
-#endif
 
-	err = fpga_mgr_buf_load(fpga_mgr, 0, data, bit_header.BitstreamLength);
+	/* struct with information about the FPGA image to program. */
+	info = fpga_image_info_alloc(&fpga_mgr->dev);
+	if (info == NULL) {
+		DRM_ERROR("%s : fpga_image_info_alloc ret NULL\n", __func__);
+		err = -ENOMEM;
+		goto free_buffers;
+	}
+
+	info->buf = data;
+	info->count = bit_header.BitstreamLength;
+	info->sgt = NULL;
+	info->firmware_name = NULL;
+	info->flags = FPGA_MGR_PARTIAL_RECONFIG;
+
+	/* Load the buffer to the FPGA */
+	err = fpga_mgr_load(fpga_mgr, info);
 	DRM_INFO("%s : ret code %d\n", __func__, err);
```

Comment:
The DRM_INFO should be replaced by DRM_ERROR on the non-zero return value of err